.changeset/thirty-ravens-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Node-based adapters now create less server-side javascript
```

Comment:
Changeset is a little awkward.

- in astro, but benefits node and vercel
- borderlines patch and minor